cjs/package.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "applicationinsights-core-js",
+    "version": "0.0.1",
+    "description": "Microsoft Application Insights Core Javascript SDK",
+    "main": "bundle/aicore.js",
+    "devDependencies": {
+        "grunt": "1.0.1",
+        "grunt-contrib-qunit": "2.0.0",
+        "grunt-contrib-uglify": "3.1.0",
+        "grunt-ts": "^6.0.0-beta.15",
+        "typescript": "2.5.3"
+    },
+    "files":[
+        "coreSDK/applicationinsights-core.*",
+        "coreSDK/JavaScriptSDK/AppInsightsCore.*",
+        "coreSDK/JavaScriptSDK.Interfaces/IAppInsightsCore.*",
+        "coreSDK/JavaScriptSDK.Interfaces/IChannelControls.*",
+        "coreSDK/JavaScriptSDK.Interfaces/ITelemetryPlugin.*",
+        "coreSDK/JavaScriptSDK.Interfaces/IConfiguration.*"
+   ],
+   "main": "coreSDK/applicationinsights-core.js",
```

Comment:
Is it okay to have two `main` entries?

Reply:
I missed taking out the first one, removing, its not needed for this file.